l10n_nl_tax_declaration_reporting/__openerp__.py

```diff
@@ -0,0 +1,45 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    Dutch accounting taxes addon
+#    Copyright (C) 2014 ONESTEiN BV (<http://www.onestein.nl>).
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+{
+    'name': 'VAT Report Netherlands',
+    'version': '1.0',
+    'category': 'Localization',
+    'sequence': 1,
+    'description': """
+NL Tax Declaration Report
+=========================
+
+This module provides you with the Tax Statement in the Dutch format.
+
+    """,
+    'author': 'ONESTEiN BV',
```

Comment:
Add author 'Odoo Community Association (OCA)' for improved visibility of the OCA on the Apps store.